assets/scss/components/_buttons.scss

```diff
@@ -0,0 +1,22 @@
+.button {
+	background-color:$color-international-orange; 
+	color:$color-white;
+	font-size:$base-font; 
+	border-radius:30px;
+	font-family:$primary-font-family; 
+	&__modal {
```

Comment:
I would have changed the class to modal__button, and put this style within the modal component